frappe/desk/reportview.py

```diff
@@ -457,14 +468,20 @@ def get_labels(fields, doctype):
 				# For example, "Item Code (Sales Invoice Item)".
 				label += f" ({ _(parenttype) })"
 
+		if aggregate_function:
+			label = _("{0} of {1}").format(aggregate_function.capitalize(), label)
+
 		labels.append(label)
 
 	return labels
 
 
 def handle_duration_fieldtype_values(doctype, data, fields):
 	for field in fields:
-		key = field.split(" as ")[0]
+		key = field
+		if " as " in key.casefold():
+			key = re.sub(" as ", " AS ", key, flags=re.IGNORECASE)
+			key = key.split(" AS ", 1)[0]
```

Comment:
```suggestion
		key = field.casefold().split(" as ", maxsplit=1)[0]
```

Is this ^ any different from what you're trying to do? Why do N iterations over field when 1 iteration do trick? :kevinface:

Same question about the block above

Reply:
>key = field.casefold().split(" as ", maxsplit=1)[0]

This makes the output lowercase

Reply:
Is there a requirement to be case sensitive?

Reply:
Not required but will have to make some more changes. Cool will do that